### DISCUSSION TITLE
Hide (****) the number of Iotas and the value of a wallet/profile

### DISCUSSION THREAD

**Discussion by Travetown** [iotaledger/firefly#1766] [Features / Enhancements]
It would be nice if we could hide the sensitive data with "***" as we did with the Hornet node.

**Comment by rafaeldjpbrochado** [comment]
Aside from the balance (and presumably the chart), what else should be hidden by this feature?

**Reply by Travetown** [reply]
<img width="772" alt="Bildschirmfoto 2021-12-17 um 08 43 43" src="https://user-images.githubusercontent.com/66588710/146508520-3ad083ae-2504-4565-872f-7203418a118f.png">

It would be nice if all the information that could be used to infer the balance could be hidden. In the community, we just had a case where an exchange requested a **VIDEO** where you had to identify the deposit address.
